Makefile

```diff
@@ -15,7 +15,10 @@ TUSB_PATH    = lib/tinyusb/src
 NRFX_PATH    = lib/nrfx
 SD_PATH      = lib/softdevice/$(SD_FILENAME)
 
-SD_VERSION   = 6.1.1
+ifndef SD_VERSION
+   SD_VERSION   = 6.1.1
+endif
+

```

Comment:
where would the `SD_VERSION` is defined for 833 ?

Reply:
Let me provide an example for the bluemicro833.

Reply:
I see, it is defined by board.mk, could you update the pca10100 as well.

Reply:
Yes.  I will also be able to test it with CircuitPython as well.

Reply:
@hathach I have a question for you. I asked @tannewt on one of his livestream and he indicated that you would be the one to ask...  The makefile for CP (nrf - see [here](https://github.com/adafruit/circuitpython/blob/main/ports/nrf/Makefile#L282)) uses a specific command to flash a bit to indicate that the application checks out and to bypass the CRC check.  Do you know if setting this bit should be done by the application UF2 or the bootloader/softdevice UF2?  Is this something new with SD7.x? Or was it there all along and the bit is set for the SD in this repo?

Reply:
@hathach you can disregard my question regarding the CRC/app check bit.  When using the 7.x booloader, all I need to do is drop the UF2 and it works perfectly.

CircuitPython loads fine with the latest build as one would expect.